src/sentry/taskworker/worker.py

```diff
@@ -1,15 +1,17 @@
 from __future__ import annotations
 
+import atexit
```

Comment:
I'm not familiar with this library. Why did we change it?

Reply:
I previously was only handling sigint, and not sigterm/sigquit.